tox.ini

```diff
@@ -0,0 +1,20 @@
+[tox]
+minversion = 3.18.0
+envlist = py38, py39, py310, py311, py312
```

Comment:
I'd just change this to py{38,39,310,311,312}

Reply:
Thanks!